### PR TITLE
Updated scripts for success WEB_VIEW registration

### PR DESCRIPTION
**Pull request by ydementieiev**
This PR is **[ready]** for review.

### Summary
Added WEB_VIEW to preloaded PT for success registration and new scripts for check reject registration. 

### CLA
- [x] I have signed [the CLA](https://docs.google.com/forms/d/e/1FAIpQLSdsgJY33VByaX482zHzi-xUm49JNnmuJOyAM6uegPQ2LXYVfA/viewform)

### CODE REVIEW

File: test_scripts/WebEngine/WebEngineProjection/001_Registration.lua [hunk 1]
```diff
@@ -33,10 +34,12 @@ config.application1.registerAppInterfaceParams.syncMsgVersion.minorVersion = 2
 --[[ Scenario ]]
 common.Title("Preconditions")
 common.Step("Clean environment", common.preconditions)
+common.Step("Update appHMIType in preloaded policy table", common.updatePreloadedPT,
+    { appSessionId, appHMIType })
```
Comment (VjKlepikov) [inline test_scripts/WebEngine/WebEngineProjection/001_Registration.lua:38]:
```suggestion
  { appSessionId, appHMIType })
```
Please remove redundant indents

File: test_scripts/WebEngine/WebEngineProjection/007_Registration_reject_WEB_VIEW_is_absent_in_PT.lua [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+--[[ General configuration parameters ]]
+config.application1.registerAppInterfaceParams.appHMIType = { appHMITypeWebView }
+config.application1.registerAppInterfaceParams.syncMsgVersion.majorVersion = 6
+config.application1.registerAppInterfaceParams.syncMsgVersion.minorVersion = 2
+
+--[[ Scenario ]]
+common.Title("Preconditions")
+common.Step("Clean environment", common.preconditions)
+common.Step("Update appHMIType in preloaded policy table", common.updatePreloadedPT,
+    { appSessionId, appHMITypeMedia })
```
Comment (VjKlepikov) [inline test_scripts/WebEngine/WebEngineProjection/007_Registration_reject_WEB_VIEW_is_absent_in_PT.lua:48]:
```suggestion
  { appSessionId, appHMITypeMedia })
```
Please remove redundant indents

File: test_scripts/WebEngine/WebEngineProjection/005_OnExitApplication_RESOURCE_CONSTRAINT_unregister.lua [hunk 3]
```diff
@@ -40,12 +44,14 @@ end
 -- [[ Scenario ]]
 common.Title("Preconditions")
 common.Step("Clean environment", common.preconditions)
+common.Step("Update appHMIType in preloaded policy table", common.updatePreloadedPT,
+    { appSessionId, appHMIType })
```
Comment (VjKlepikov) [inline test_scripts/WebEngine/WebEngineProjection/005_OnExitApplication_RESOURCE_CONSTRAINT_unregister.lua:48]:
```suggestion
  { appSessionId, appHMIType })
```
Please remove redundant indents

File: test_scripts/WebEngine/WebEngineProjection/004_Check_that_SDL_forwards_OnSystemCapabilityUpdated_notification_to_WebEngine_projection_app.lua [hunk 4]
```diff
@@ -26,6 +26,7 @@ config.defaultMobileAdapterType = "WS"
 common.testSettings.restrictions.sdlBuildOptions = {{ webSocketServerSupport = {"ON"} }}
```
Comment (VjKlepikov) [inline test_scripts/WebEngine/WebEngineProjection/004_Check_that_SDL_forwards_OnSystemCapabilityUpdated_notification_to_WebEngine_projection_app.lua:26]:
```suggestion
common.testSettings.restrictions.sdlBuildOptions = {{ webSocketServerSupport = { "ON" }}}
```
 Please add a space

File: test_scripts/WebEngine/WebEngineProjection/002_Check_WS_connection_for_WebEngine_projection_app_in_NONE.lua [hunk 5]
```diff
@@ -46,14 +47,15 @@ end
 common.Title("Preconditions")
 common.Step("Clean environment", common.preconditions)
 common.Step("Update WS Server Certificate parameters in smartDeviceLink.ini file", common.commentAllCertInIniFile)
+common.Step("Update appHMIType in preloaded policy table", common.updatePreloadedPT, { appSessionId, appHMIType})
```
Comment (VjKlepikov) [inline test_scripts/WebEngine/WebEngineProjection/002_Check_WS_connection_for_WebEngine_projection_app_in_NONE.lua:50]:
```suggestion
common.Step("Update appHMIType in preloaded policy table", common.updatePreloadedPT, { appSessionId, appHMIType })
```
 Please add a space

File: test_scripts/WebEngine/WebEngineProjection/001_Registration.lua (hunk 1 above)
Comment (ydementieiev) [reply test_scripts/WebEngine/WebEngineProjection/001_Registration.lua]: @VjKlepikov [done](https://github.com/LuxoftSDL/sdl_atf_test_scripts/pull/4/commits/e1b8a7c00153273c0320dc9b9b19fe8f2c9a6410#diff-4f3559029bee4e49c37fb96dd1163cc6R38)

File: test_scripts/WebEngine/WebEngineProjection/007_Registration_reject_WEB_VIEW_is_absent_in_PT.lua (hunk 2 above)
Comment (ydementieiev) [reply test_scripts/WebEngine/WebEngineProjection/007_Registration_reject_WEB_VIEW_is_absent_in_PT.lua]: @VjKlepikov [done](https://github.com/LuxoftSDL/sdl_atf_test_scripts/pull/4/commits/e1b8a7c00153273c0320dc9b9b19fe8f2c9a6410#diff-82aaf8dae27f1b695f9cc2b4592ceba3R48)

File: test_scripts/WebEngine/WebEngineProjection/005_OnExitApplication_RESOURCE_CONSTRAINT_unregister.lua (hunk 3 above)
Comment (ydementieiev) [reply test_scripts/WebEngine/WebEngineProjection/005_OnExitApplication_RESOURCE_CONSTRAINT_unregister.lua]: @VjKlepikov [done](https://github.com/LuxoftSDL/sdl_atf_test_scripts/pull/4/commits/e1b8a7c00153273c0320dc9b9b19fe8f2c9a6410#diff-d8e481ca736c621d5cf917740b5d76f7R48)

File: test_scripts/WebEngine/WebEngineProjection/004_Check_that_SDL_forwards_OnSystemCapabilityUpdated_notification_to_WebEngine_projection_app.lua (hunk 4 above)
Comment (ydementieiev) [reply test_scripts/WebEngine/WebEngineProjection/004_Check_that_SDL_forwards_OnSystemCapabilityUpdated_notification_to_WebEngine_projection_app.lua]: @VjKlepikov [done](https://github.com/LuxoftSDL/sdl_atf_test_scripts/pull/4/commits/e1b8a7c00153273c0320dc9b9b19fe8f2c9a6410#diff-c1be9f63e43075846d073f23fb506234R26)

File: test_scripts/WebEngine/WebEngineProjection/002_Check_WS_connection_for_WebEngine_projection_app_in_NONE.lua (hunk 5 above)
Comment (ydementieiev) [reply test_scripts/WebEngine/WebEngineProjection/002_Check_WS_connection_for_WebEngine_projection_app_in_NONE.lua]: @VjKlepikov [done](https://github.com/LuxoftSDL/sdl_atf_test_scripts/pull/4/commits/e1b8a7c00153273c0320dc9b9b19fe8f2c9a6410#diff-683efc5105232f444af5b1cb634b7da7R50)

File: test_scripts/WebEngine/commonWebEngine.lua [hunk 6]
```diff
@@ -99,6 +101,15 @@ function common.validation(actualData, expectedData, pMessage)
   return true
 end
 
+function common.updatePreloadedPT(pAppId, appHMIType)
+  local preloadedTable = common.getPreloadedPT()
+  local appId = config["application" .. pAppId].registerAppInterfaceParams.fullAppID
+  preloadedTable.policy_table.app_policies[appId] = common.cloneTable(preloadedTable.policy_table.app_policies.default)
+  preloadedTable.policy_table.app_policies[appId].AppHMIType = { appHMIType }
```
Comment (VjKlepikov) [inline test_scripts/WebEngine/commonWebEngine.lua:108]:
```suggestion
  preloadedTable.policy_table.app_policies[appId].AppHMIType = { appHMIType }
  preloadedTable.policy_table.app_policies[appId].enabled = true
```
Add mandatory parameter for WebEngine app

File: test_scripts/WebEngine/WebEngineProjection/006_Registration_reject_WEB_VIEW_when_PT_is_empty.lua [hunk 7]
```diff
@@ -0,0 +1,53 @@
...
+--  Performed steps
+--  appID->RegisterAppInterface(params)
+--
+--  Expected behavior:
+--     Policy table is empty.
+--     SDL->appID: REJECT, success:"false":RegisterAppInterface()
+---------------------------------------------------------------------------------------------------
+-- [[ Required Shared Libraries ]]
+local common = require('test_scripts/WebEngine/commonWebEngine')
+
```
Comment (VjKlepikov) [inline test_scripts/WebEngine/WebEngineProjection/006_Registration_reject_WEB_VIEW_when_PT_is_empty.lua:22]:
```suggestion
--[[ Test Configuration ]]
config.defaultMobileAdapterType = "WS"
common.testSettings.restrictions.sdlBuildOptions = {{ webSocketServerSupport = { "ON" }}}
```
Update test to check webEngine app over the WebSocket connection

File: test_scripts/WebEngine/WebEngineProjection/006_Registration_reject_WEB_VIEW_when_PT_is_empty.lua [hunk 8]
```diff
@@ -0,0 +1,53 @@
...
+--[[ General configuration parameters ]]
+config.application1.registerAppInterfaceParams.appHMIType = { appHMIType }
+config.application1.registerAppInterfaceParams.syncMsgVersion.majorVersion = 6
+config.application1.registerAppInterfaceParams.syncMsgVersion.minorVersion = 2
+
+--[[ Scenario ]]
+common.Title("Preconditions")
+common.Step("Clean environment", common.preconditions)
+common.Step("Start SDL, HMI, connect Mobile", common.start)
```
Comment (VjKlepikov) [inline test_scripts/WebEngine/WebEngineProjection/006_Registration_reject_WEB_VIEW_when_PT_is_empty.lua:46]:
```suggestion
common.Step("Start SDL, HMI", common.startWOdeviceConnect)
common.Step("Connect WebEngine device", common.connectWebEngine, { appSessionId, config.defaultMobileAdapterType })
```

File: test_scripts/WebEngine/WebEngineProjection/007_Registration_reject_WEB_VIEW_is_absent_in_PT.lua [hunk 9]
```diff
@@ -0,0 +1,56 @@
...
+--  Performed steps
+--  appID->RegisterAppInterface(params)
+--
+--  Expected behavior:
+--     Policy table not contain WEB_VIEW.
+--     SDL->appID: REJECT, success:"false":RegisterAppInterface()
+---------------------------------------------------------------------------------------------------
+-- [[ Required Shared Libraries ]]
+local common = require('test_scripts/WebEngine/commonWebEngine')
+
```
Comment (VjKlepikov) [inline test_scripts/WebEngine/WebEngineProjection/007_Registration_reject_WEB_VIEW_is_absent_in_PT.lua:22]:
```suggestion
config.defaultMobileAdapterType = "WS"
common.testSettings.restrictions.sdlBuildOptions = {{ webSocketServerSupport = { "ON" }}}
```
Update test to check webEngine app over the WebSocket connection

File: test_scripts/WebEngine/commonWebEngine.lua [hunk 10]
```diff
@@ -99,6 +101,15 @@ function common.validation(actualData, expectedData, pMessage)
   return true
 end
 
+function common.updatePreloadedPT(pAppId, appHMIType)
+  local preloadedTable = common.getPreloadedPT()
+  local appId = config["application" .. pAppId].registerAppInterfaceParams.fullAppID
+  preloadedTable.policy_table.app_policies[appId] = common.cloneTable(preloadedTable.policy_table.app_policies.default)
```
Comment (LitvinenkoIra) [inline test_scripts/WebEngine/commonWebEngine.lua:107]: @ydementieiev Please check if `enabled = true ` parameter is present in the policy db. This parameter should be present for WebEngine app

File: test_scripts/WebEngine/WebEngineProjection/007_Registration_reject_WEB_VIEW_is_absent_in_PT.lua [hunk 11]
```diff
@@ -0,0 +1,56 @@
...
+config.application1.registerAppInterfaceParams.appHMIType = { appHMITypeWebView }
+config.application1.registerAppInterfaceParams.syncMsgVersion.majorVersion = 6
+config.application1.registerAppInterfaceParams.syncMsgVersion.minorVersion = 2
+
+--[[ Scenario ]]
+common.Title("Preconditions")
+common.Step("Clean environment", common.preconditions)
+common.Step("Update appHMIType in preloaded policy table", common.updatePreloadedPT,
+  { appSessionId, appHMITypeMedia })
+common.Step("Start SDL, HMI, connect Mobile", common.start)
```
Comment (VjKlepikov) [inline test_scripts/WebEngine/WebEngineProjection/007_Registration_reject_WEB_VIEW_is_absent_in_PT.lua:49]:
```suggestion
common.Step("Start SDL, HMI", common.startWOdeviceConnect)
common.Step("Connect WebEngine device", common.connectWebEngine, { appSessionId, config.defaultMobileAdapterType })
```

File: test_scripts/WebEngine/WebEngineProjection/001_Registration.lua [hunk 12]
```diff
@@ -23,6 +23,7 @@
 local common = require('test_scripts/WebEngine/commonWebEngine')
```
Comment (VjKlepikov) [inline test_scripts/WebEngine/WebEngineProjection/001_Registration.lua:24]:
```suggestion
config.defaultMobileAdapterType = "WS"
common.testSettings.restrictions.sdlBuildOptions = {{ webSocketServerSupport = { "ON" }}}
```
Update test to check webEngine app over the WebSocket connection

File: test_scripts/WebEngine/WebEngineProjection/001_Registration.lua [hunk 13]
```diff
@@ -33,10 +34,12 @@ config.application1.registerAppInterfaceParams.syncMsgVersion.minorVersion = 2
 --[[ Scenario ]]
 common.Title("Preconditions")
 common.Step("Clean environment", common.preconditions)
+common.Step("Update appHMIType in preloaded policy table", common.updatePreloadedPT,
+  { appSessionId, appHMIType })
 common.Step("Start SDL, HMI, connect Mobile", common.start)
```
Comment (VjKlepikov) [inline test_scripts/WebEngine/WebEngineProjection/001_Registration.lua:39]:
```suggestion
common.Step("Start SDL, HMI", common.startWOdeviceConnect)
common.Step("Connect WebEngine device", common.connectWebEngine, { appSessionId, config.defaultMobileAdapterType })
```

File: test_scripts/WebEngine/commonWebEngine.lua (hunk 6 above)
Comment (ydementieiev) [reply test_scripts/WebEngine/commonWebEngine.lua:108]: @VjKlepikov [done](https://github.com/LuxoftSDL/sdl_atf_test_scripts/pull/4/commits/0f412587275cf5245150b036bc28189967421e9c#diff-7b7ca1c3b1b899ba16ded7d51adceea0R109)

File: test_scripts/WebEngine/WebEngineProjection/006_Registration_reject_WEB_VIEW_when_PT_is_empty.lua (hunk 7 above)
Comment (ydementieiev) [reply test_scripts/WebEngine/WebEngineProjection/006_Registration_reject_WEB_VIEW_when_PT_is_empty.lua:22]: @VjKlepikov [done](https://github.com/LuxoftSDL/sdl_atf_test_scripts/pull/4/commits/0f412587275cf5245150b036bc28189967421e9c#diff-ff5652b3ded34af081249da10ca6203fR23)

File: test_scripts/WebEngine/WebEngineProjection/006_Registration_reject_WEB_VIEW_when_PT_is_empty.lua (hunk 8 above)
Comment (ydementieiev) [reply test_scripts/WebEngine/WebEngineProjection/006_Registration_reject_WEB_VIEW_when_PT_is_empty.lua]: @VjKlepikov [done](https://github.com/LuxoftSDL/sdl_atf_test_scripts/pull/4/commits/0f412587275cf5245150b036bc28189967421e9c#diff-ff5652b3ded34af081249da10ca6203fR50)

File: test_scripts/WebEngine/WebEngineProjection/007_Registration_reject_WEB_VIEW_is_absent_in_PT.lua (hunk 9 above)
Comment (ydementieiev) [reply test_scripts/WebEngine/WebEngineProjection/007_Registration_reject_WEB_VIEW_is_absent_in_PT.lua:22]: @VjKlepikov [done](https://github.com/LuxoftSDL/sdl_atf_test_scripts/pull/4/commits/0f412587275cf5245150b036bc28189967421e9c#diff-82aaf8dae27f1b695f9cc2b4592ceba3R24)

File: test_scripts/WebEngine/commonWebEngine.lua (hunk 10 above)
Comment (ydementieiev) [reply test_scripts/WebEngine/commonWebEngine.lua:107]: @LitvinenkoIra [added](https://github.com/LuxoftSDL/sdl_atf_test_scripts/pull/4/commits/0f412587275cf5245150b036bc28189967421e9c#diff-7b7ca1c3b1b899ba16ded7d51adceea0R109) enable = true

File: test_scripts/WebEngine/WebEngineProjection/007_Registration_reject_WEB_VIEW_is_absent_in_PT.lua (hunk 11 above)
Comment (ydementieiev) [reply test_scripts/WebEngine/WebEngineProjection/007_Registration_reject_WEB_VIEW_is_absent_in_PT.lua]: @VjKlepikov [done](https://github.com/LuxoftSDL/sdl_atf_test_scripts/pull/4/commits/0f412587275cf5245150b036bc28189967421e9c#diff-82aaf8dae27f1b695f9cc2b4592ceba3R53)

File: test_scripts/WebEngine/WebEngineProjection/001_Registration.lua (hunk 12 above)
Comment (ydementieiev) [reply test_scripts/WebEngine/WebEngineProjection/001_Registration.lua:24]: @VjKlepikov [done](https://github.com/LuxoftSDL/sdl_atf_test_scripts/pull/4/commits/0f412587275cf5245150b036bc28189967421e9c#diff-4f3559029bee4e49c37fb96dd1163cc6R26)

File: test_scripts/WebEngine/WebEngineProjection/001_Registration.lua (hunk 13 above)
Comment (ydementieiev) [reply test_scripts/WebEngine/WebEngineProjection/001_Registration.lua]: @VjKlepikov [done](https://github.com/LuxoftSDL/sdl_atf_test_scripts/pull/4/commits/0f412587275cf5245150b036bc28189967421e9c#diff-4f3559029bee4e49c37fb96dd1163cc6R43)

File: test_scripts/WebEngine/WebEngineProjection/005_OnExitApplication_RESOURCE_CONSTRAINT_unregister.lua [hunk 14]
```diff
@@ -13,16 +13,20 @@
 --  a. Web app is unregistered
 --  b. Mobile session is disconnected
 ---------------------------------------------------------------------------------------------------
+--[[ Required Shared libraries ]]
```
Comment (aderiabin) [inline test_scripts/WebEngine/WebEngineProjection/005_OnExitApplication_RESOURCE_CONSTRAINT_unregister.lua:16]: @ydementieiev Please rewrite test script description in one style in all scripts of this PR. I would propose to use description style which is used in this script.

File: test_scripts/WebEngine/WebEngineProjection/006_Registration_reject_WEB_VIEW_when_PT_is_empty.lua [hunk 15]
```diff
@@ -0,0 +1,59 @@
...
+local appSessionId = 1
+local appHMIType = "WEB_VIEW"
+
+--[[ Local Functions ]]
+local function registerApp()
+  local session = common.createSession(appSessionId)
+  session:StartService(7)
+  :Do(function()
+      local corId = session:SendRPC("RegisterAppInterface", common.getParams(appSessionId))
+      EXPECT_HMINOTIFICATION("BasicCommunication.OnAppRegistered"):Times(0)
```
Comment (aderiabin) [inline test_scripts/WebEngine/WebEngineProjection/006_Registration_reject_WEB_VIEW_when_PT_is_empty.lua:37]: @ydementieiev Please avoid to use general expectation functions like `EXPECT_HMINOTIFICATION()` they are deprecated. Use `actions.hmi.getConnection():ExpectNotification()` instead.
```suggestion
      common.getHMIConnection:ExpectNotification("BasicCommunication.OnAppRegistered"):Times(0)
```

File: test_scripts/WebEngine/WebEngineProjection/006_Registration_reject_WEB_VIEW_when_PT_is_empty.lua [hunk 16]
```diff
@@ -0,0 +1,59 @@
...
+--[[ Test Configuration ]]
+config.defaultMobileAdapterType = "WS"
+common.testSettings.restrictions.sdlBuildOptions = {{ webSocketServerSupport = { "ON" }}}
+
+--[[ Local Variables ]]
+local appSessionId = 1
+local appHMIType = "WEB_VIEW"
+
+--[[ Local Functions ]]
+local function registerApp()
```
Comment (aderiabin) [inline test_scripts/WebEngine/WebEngineProjection/006_Registration_reject_WEB_VIEW_when_PT_is_empty.lua:32]: @ydementieiev Please move this function into common module (it is used by more than one script) and rename it to better reflect its purpose.

File: test_scripts/WebEngine/WebEngineProjection/006_Registration_reject_WEB_VIEW_when_PT_is_empty.lua [hunk 17]
```diff
@@ -0,0 +1,59 @@
+--  Requirement summary:
+--  [RegisterAppInterface] REJECT
+--  [RegisterAppInterface] RegisterAppInterface and HMILevel
+--
+--  Description:
+--  Check that App will be rejected with HMI type WEB_VIEW 
+--  when policy table is empty.
```
Comment (aderiabin) [inline test_scripts/WebEngine/WebEngineProjection/006_Registration_reject_WEB_VIEW_when_PT_is_empty.lua:7]: @ydementieiev This description of test script is incorrect. SDL can not start with empty policy table.
I think you mean that mobile application has no policy record in local policy table here. 
Please correct test script description.

File: test_scripts/WebEngine/WebEngineProjection/005_OnExitApplication_RESOURCE_CONSTRAINT_unregister.lua (hunk 14 above)
Comment (ydementieiev) [reply test_scripts/WebEngine/WebEngineProjection/005_OnExitApplication_RESOURCE_CONSTRAINT_unregister.lua:16]: @aderiabin [done](https://github.com/LuxoftSDL/sdl_atf_test_scripts/pull/4/commits/9271d45b0b4e176fad4cd4070eeb060facb11f43) look please

File: test_scripts/WebEngine/WebEngineProjection/006_Registration_reject_WEB_VIEW_when_PT_is_empty.lua (hunk 15 above)
Comment (ydementieiev) [reply test_scripts/WebEngine/WebEngineProjection/006_Registration_reject_WEB_VIEW_when_PT_is_empty.lua]: @aderiabin [changed](https://github.com/LuxoftSDL/sdl_atf_test_scripts/pull/4/commits/9271d45b0b4e176fad4cd4070eeb060facb11f43#diff-7b7ca1c3b1b899ba16ded7d51adceea0R119)

File: test_scripts/WebEngine/WebEngineProjection/006_Registration_reject_WEB_VIEW_when_PT_is_empty.lua (hunk 16 above)
Comment (ydementieiev) [reply test_scripts/WebEngine/WebEngineProjection/006_Registration_reject_WEB_VIEW_when_PT_is_empty.lua]: @aderiabin [moved](https://github.com/LuxoftSDL/sdl_atf_test_scripts/pull/4/commits/9271d45b0b4e176fad4cd4070eeb060facb11f43#diff-7b7ca1c3b1b899ba16ded7d51adceea0R119)

File: test_scripts/WebEngine/WebEngineProjection/006_Registration_reject_WEB_VIEW_when_PT_is_empty.lua (hunk 17 above)
Comment (ydementieiev) [reply test_scripts/WebEngine/WebEngineProjection/006_Registration_reject_WEB_VIEW_when_PT_is_empty.lua]: @aderiabin thanks you ! [done ](https://github.com/LuxoftSDL/sdl_atf_test_scripts/pull/4/commits/9271d45b0b4e176fad4cd4070eeb060facb11f43#diff-ff5652b3ded34af081249da10ca6203fR7)

File: test_scripts/WebEngine/WebEngineProjection/001_Registration.lua [hunk 18]
```diff
@@ -23,20 +23,27 @@
...
 config.application1.registerAppInterfaceParams.syncMsgVersion.majorVersion = 6
 config.application1.registerAppInterfaceParams.syncMsgVersion.minorVersion = 2
+config.defaultMobileAdapterType = "WS"
+common.testSettings.restrictions.sdlBuildOptions = {{ webSocketServerSupport = { "ON" }}}
 
 --[[ Scenario ]]
 common.Title("Preconditions")
 common.Step("Clean environment", common.preconditions)
-common.Step("Start SDL, HMI, connect Mobile", common.start)
+common.Step("Update appHMIType in preloaded policy table", common.updatePreloadedPT,
```
Comment (AByzhynar) [inline test_scripts/WebEngine/WebEngineProjection/001_Registration.lua:39]: @ydementieiev I would recommend to change it to `Update AppHMIType in preloaded policy table` as in PT there is no such field like `appHMIType` but `AppHMIType`  is valid one. This is related to all scripts where you add AppHMIType to PT.

File: test_scripts/WebEngine/WebEngineProjection/007_Registration_reject_WEB_VIEW_is_absent_in_PT.lua [hunk 19]
```diff
@@ -0,0 +1,49 @@
...
+common.Title("Preconditions")
+common.Step("Clean environment", common.preconditions)
+common.Step("Update appHMIType in preloaded policy table", common.updatePreloadedPT,
+  { appSessionId, appHMITypeMedia })
+common.Step("Start SDL, HMI", common.startWOdeviceConnect)
+common.Step("Connect WebEngine device", common.connectWebEngine,
+  { appSessionId, config.defaultMobileAdapterType })
+
+common.Title("Test")
+common.Step("Register App, policy table not contain WEB_VIEW", common.rejectedRegisterApp)
```
Comment (AByzhynar) [inline test_scripts/WebEngine/WebEngineProjection/007_Registration_reject_WEB_VIEW_is_absent_in_PT.lua:45]: @ydementieiev `PT does not contain "WEB_VIEW" AppHMIType`

File: test_scripts/WebEngine/WebEngineProjection/001_Registration.lua (hunk 18 above)
Comment (ydementieiev) [reply test_scripts/WebEngine/WebEngineProjection/001_Registration.lua]: @AByzhynar thank you, [changed](https://github.com/LuxoftSDL/sdl_atf_test_scripts/pull/4/commits/abede64e2119bad9c40c14ecd9e5ef9e03ce0228)

File: test_scripts/WebEngine/WebEngineProjection/007_Registration_reject_WEB_VIEW_is_absent_in_PT.lua (hunk 19 above)
Comment (ydementieiev) [reply test_scripts/WebEngine/WebEngineProjection/007_Registration_reject_WEB_VIEW_is_absent_in_PT.lua]: @AByzhynar thank you, [changed](https://github.com/LuxoftSDL/sdl_atf_test_scripts/pull/4/commits/abede64e2119bad9c40c14ecd9e5ef9e03ce0228)

File: test_scripts/WebEngine/WebEngineProjection/006_Registration_reject_WEB_VIEW_when_PT_is_empty.lua [hunk 20]
```diff
@@ -0,0 +1,46 @@
...
+--
+--  Used precondition
+--  SDL, HMI are running on system.
+--  Mobile device is connected to SDL.
+--
+--  Performed steps
+--  appID->RegisterAppInterface(params)
+--
+--  Expected behavior:
+--     Policy table is empty.
```
Comment (aderiabin) [inline test_scripts/WebEngine/WebEngineProjection/006_Registration_reject_WEB_VIEW_when_PT_is_empty.lua:17]: @ydementieiev Really strange expectation as for me. Why SDL must clear LPT on registration of application?
I think it is make sense to check that SDL do not add any permissions into LPT for application which tried to register here. 
But it requires to extend test scenario with new test step (triggfer PTU and check policy permissions for application which tried to register in PTS)

File: test_scripts/WebEngine/WebEngineProjection/007_Registration_reject_WEB_VIEW_is_absent_in_PT.lua [hunk 21]
```diff
@@ -0,0 +1,49 @@
...
+--
+--  Used precondition
+--  SDL, HMI are running on system.
+--  Mobile device is connected to SDL.
+--
+--  Performed steps
+--  appID->RegisterAppInterface(params)
+--
+--  Expected behavior:
+--     Policy table not contain WEB_VIEW.
```
Comment (aderiabin) [inline test_scripts/WebEngine/WebEngineProjection/007_Registration_reject_WEB_VIEW_is_absent_in_PT.lua:17]: @ydementieiev This expectation is redundant, please remove it.

File: test_scripts/WebEngine/WebEngineProjection/006_Registration_reject_WEB_VIEW_when_PT_is_empty.lua (hunk 20 above)
Comment (ydementieiev) [reply test_scripts/WebEngine/WebEngineProjection/006_Registration_reject_WEB_VIEW_when_PT_is_empty.lua]: @aderiabin check please [changes ](https://github.com/LuxoftSDL/sdl_atf_test_scripts/pull/4/commits/33588521b5f1ff45678b21bd871324afc6f42bd0)

File: test_scripts/WebEngine/WebEngineProjection/007_Registration_reject_WEB_VIEW_is_absent_in_PT.lua (hunk 21 above)
Comment (ydementieiev) [reply test_scripts/WebEngine/WebEngineProjection/007_Registration_reject_WEB_VIEW_is_absent_in_PT.lua]: @aderiabin reworked [description](https://github.com/LuxoftSDL/sdl_atf_test_scripts/pull/4/commits/33588521b5f1ff45678b21bd871324afc6f42bd0)

File: test_scripts/WebEngine/WebEngineProjection/001_Registration.lua [hunk 22]
```diff
@@ -1,42 +1,42 @@
...
 
 --[[ Scenario ]]
 common.Title("Preconditions")
 common.Step("Clean environment", common.preconditions)
-common.Step("Start SDL, HMI, connect Mobile", common.start)
+common.Step("Add AppHMIType to preloaded policy table", common.updatePreloadedPT,
+  { appSessionId, appHMIType })
+common.Step("Start SDL, HMI", common.startWOdeviceConnect)
+common.Step("Connect WebEngine device", common.connectWebEngine,
+  { appSessionId, config.defaultMobileAdapterType })
```
Comment (aderiabin) [inline test_scripts/WebEngine/WebEngineProjection/001_Registration.lua:36]: @ydementieiev It is enough to set `config.defaultMobileAdapterType = "WS"` that is done already.
```suggestion
common.Step("Start SDL, HMI, connect Mobile", common.start)
```

File: test_scripts/WebEngine/WebEngineProjection/007_Registration_reject_WEB_VIEW_is_absent_in_PT.lua [hunk 23]
```diff
@@ -0,0 +1,45 @@
...
+common.testSettings.restrictions.sdlBuildOptions = {{ webSocketServerSupport = { "ON" }}}
+
+--[[ Scenario ]]
+common.Title("Preconditions")
+common.Step("Clean environment", common.preconditions)
+common.Step("Add AppHMIType to preloaded policy table", common.updatePreloadedPT,
+  { appSessionId, appHMITypeMedia })
+common.Step("Start SDL, HMI", common.startWOdeviceConnect)
+common.Step("Connect WebEngine device", common.connectWebEngine,
+  { appSessionId, config.defaultMobileAdapterType })
```
Comment (aderiabin) [inline test_scripts/WebEngine/WebEngineProjection/007_Registration_reject_WEB_VIEW_is_absent_in_PT.lua:38]: @ydementieiev It is enough to set `config.defaultMobileAdapterType = "WS"` that is done already.
```suggestion
common.Step("Start SDL, HMI, connect Mobile", common.start)
```

File: test_scripts/WebEngine/commonWebEngine.lua [hunk 24]
```diff
@@ -99,6 +102,26 @@ function common.validation(actualData, expectedData, pMessage)
   return true
 end
 
+function common.updatePreloadedPT(pAppId, pAppHMIType)
+  local preloadedTable = common.getPreloadedPT()
+  local appId = config["application" .. pAppId].registerAppInterfaceParams.fullAppID
+  preloadedTable.policy_table.app_policies[appId] = common.cloneTable(preloadedTable.policy_table.app_policies.default)
+  preloadedTable.policy_table.app_policies[appId].AppHMIType = { pAppHMIType }
+  preloadedTable.policy_table.app_policies[appId].enabled = true
+  preloadedTable.policy_table.functional_groupings["DataConsent-2"].rpcs = common.null
```
Comment (aderiabin) [inline test_scripts/WebEngine/commonWebEngine.lua:111]: @ydementieiev It is to add only `enabled` flag is not enough to set webengine application in policy table.
```suggestion
  local appPermissions = common.cloneTable(preloadedTable.policy_table.app_policies.default)
  appPermissions.AppHMIType = { pAppHMIType }
  appPermissions.enabled = true
  appPermissions.transportType = "WS",
  appPermissions.hybridAppPreference = "CLOUD"
  preloadedTable.policy_table.app_policies[appId] = appPermissions
  preloadedTable.policy_table.functional_groupings["DataConsent-2"].rpcs = common.null
```

Comment (ydementieiev) [reply test_scripts/WebEngine/commonWebEngine.lua]: @aderiabin thank you , done

File: test_scripts/WebEngine/WebEngineProjection/007_Registration_reject_WEB_VIEW_is_absent_in_PT.lua (hunk 23 above)
Comment (ydementieiev) [reply test_scripts/WebEngine/WebEngineProjection/007_Registration_reject_WEB_VIEW_is_absent_in_PT.lua]: @aderiabin thank you , done

File: test_scripts/WebEngine/WebEngineProjection/001_Registration.lua (hunk 22 above)
Comment (ydementieiev) [reply test_scripts/WebEngine/WebEngineProjection/001_Registration.lua]: @aderiabin thank you , done

**Pull request merged**